dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.10-slim-buster
+
+ENV TZ="America/Chicago"
```

Comment:
Can you add a note to the README that the timezone needs to be set in the dockerfile if deploying with Docker?

Reply:
Added a note in the readme - I could add a .env and instructions with docker build if you want that much detail too